src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -122,6 +122,9 @@ public static Builder builder(Class<?> type) {
   public Builder toBuilder() {
     Builder builder = new Builder(type);
     builder.members.putAll(members);
+    for (Map.Entry<String, List<CodeBlock>> entry : builder.members.entrySet()) {
```

Comment:
Regarding the `mutableMultimap()`: then, one has to either remove the `final` modifier from Builder#members or pass the target map as a second parameter to the utility method. Both are not so nice solutions, imho. What do you think? Do it inline as is, and iterate over members' entry set.